library/psa_crypto.c

```diff
@@ -3499,32 +3499,38 @@ uint32_t mbedtls_psa_interruptible_get_max_ops( void )
 }
 
 uint32_t mbedtls_psa_sign_hash_get_num_ops(
-            const mbedtls_psa_sign_hash_interruptible_operation_t *operation )
+                    mbedtls_psa_sign_hash_interruptible_operation_t *operation)
```

Comment:
I don't understand why this is not `const`.

Well, I kind of do given how it's implemented, but:

1. Why update `num_ops` when the application queries it, instead of when `complete` returns?
2. This doesn't work anyway, see my comment below.